source/documentation/standards/naming-domains.html.md.erb

```diff
@@ -97,3 +109,11 @@ You should follow the above advice for these sorts of reasons:
 ![deceptive-site-alert](images/deceptive-site-alert.png)
 
 Whilst we can get Google to remove the flag, it will keep happening.
+
+## Deprecating domains within the MOJ
+
+To reduce our reliance on non `.gov.uk` domains, the following are no longer acceptable to use for new services:
+
+- `dsd.io` subdomains
```

Comment:
Maybe also offer some context for this, and why it's no longer good?

Something like:

> Historically used for staff services, development/test environments, and internal tools, but suffered all of the <pitfalls of non GOV.UK domains>